*** sudoers     2010-04-18 11:12:22.000000000 -0700
--- sudoers2    2013-01-03 09:38:25.000000000 -0800
***************
*** 83,89 ****
  # %wheel      ALL=(ALL)       ALL
  
  ## Same thing without a password
! # %wheel      ALL=(ALL)       NOPASSWD: ALL
  
  ## Allows members of the users group to mount and unmount the 
  ## cdrom as root
--- 83,89 ----
  # %wheel      ALL=(ALL)       ALL
  
  ## Same thing without a password
! %wheel        ALL=(ALL)       NOPASSWD: ALL
  
  ## Allows members of the users group to mount and unmount the 
  ## cdrom as root
